spiped/main.c

```diff
@@ -15,6 +15,8 @@
 #include "dispatch.h"
 #include "proto_crypt.h"
 
+volatile int should_shutdown = 0;
```

Comment:
Is it `volatile sig_atomic_t` or just `sig_atomic_t`?
Am 14.05.2016 23:31 schrieb "Colin Percival" <notifications@github.com>:

> In spiped/main.c
> <https://github.com/Tarsnap/spiped/pull/51#discussion_r63282337>:
>
> > @@ -15,6 +15,8 @@
> >  #include "dispatch.h"
> >  #include "proto_crypt.h"
> >
> > +volatile int should_shutdown = 0;
>
> sig_atomic_t
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Tarsnap/spiped/pull/51/files/b760ac699dd7f655055afe5840f27b44c578da81#r63282337>
>